### DISCUSSION TITLE
why do not prompt user account name last used?

### DISCUSSION THREAD

**Discussion by sibosend** [Kong/insomnia#7264] [Feedback]
first of all, this tool is great, and secondly, i understand why need user to log in. But can you remind the user what the account name was used last time? Now after logged in again,  i lost my test  configured ......

**Comment by subnetmarco** [comment]
> Now after logged in again, i lost my test configured

Just so I understand, you were able to login again using the same account and the test was lost? Or you logged in with a different account (therefore a different email address or SSO login) and therefore the previous data is not visible anymore because it belongs to a different account?

**Reply by sibosend** [reply]
thanks for your reply.  for my scenario, i may logged in with different account (so many accounts ...)
